Data-Sources.md

```diff
@@ -2,5 +2,53 @@
 
 This section will describe **Data Source** management in Cacti.
 
+Data sources in cacti are the points of data that cacti will collect from a device here are some examples
+
+You have a ping monitor going to a device that ping monitor will count as 1 data source.
+
+If you have a 24 port switch and you poll the device via snmp and graph all of the ports then there will be 24 data sources
+However if you add more graphs that base their data on the original data source that would not count as another data source
```

Comment:
There is no gap between this and the previous paragraph.  Without one, they will be both merged into a single paragraph when the markdown is parsed.